build/release.mk

```diff
@@ -0,0 +1,42 @@
+BUILD_DIR    := ./bin/
+GORELEASER_VERSION := v0.155.0
+GORELEASER_BIN ?= bin/goreleaser
+
+bin:
+	@mkdir -p $(BUILD_DIR)
+
+$(GORELEASER_BIN): bin
+	@echo "=== [$(GORELEASER_BIN)] Installing goreleaser $(GORELEASER_VERSION)"
+	@(wget -qO /tmp/goreleaser.tar.gz https://github.com/goreleaser/goreleaser/releases/download/$(GORELEASER_VERSION)/goreleaser_$(OS_DOWNLOAD)_x86_64.tar.gz)
+	@(tar -xf  /tmp/goreleaser.tar.gz -C bin/)
+	@(rm -f /tmp/goreleaser.tar.gz)
+	@echo "=== [$(GORELEASER_BIN)] goreleaser downloaded"
+
+.PHONY : release/clean
+release/clean:
+	@echo "=== [release/clean] remove build metadata files"
+	rm -fv $(CURDIR)/cmd/newrelic-infra/versioninfo.json
+	rm -fv $(CURDIR)/cmd/newrelic-infra/resource.syso
+
+.PHONY : release/deps
+release/deps: $(GORELEASER_BIN)
+	@echo "=== [release/deps] install goversioninfo"
+	@go get github.com/josephspurrier/goversioninfo/cmd/goversioninfo
```

Comment:
we have this here as well https://github.com/newrelic/infrastructure-agent/pull/282/files#diff-0b1289ef866dadf79c60612a2c21e5bba3dc7d123136b4e97d547f5b899da075R32

can't we manage this from go mod?
we do that for non-production (ie: development/CI) dependencies

Reply:
I've removed the installation of goversion info as its only required for windows